web/models/user.ex

```diff
@@ -103,12 +103,6 @@ defmodule CodeCorps.User do
     |> apply_state_transition(struct)
   end
 
-  def github_association_changeset(struct, params) do
-    struct
-    |> cast(params, [:github_auth_token, :github_avatar_url, :github_email, :github_id, :github_username])
-    |> validate_required([:github_auth_token, :github_avatar_url, :github_id, :github_username])
-  end
```

Comment:
No need to expose it separately, since it's a simple thing. Should be enough to test the module calling this behaviour.